docs/clearlinux-installation-guide.md

```diff
@@ -0,0 +1,52 @@
+# Installing Clear Containers 3.0 on Clear Linux
+
+Clear Containers 3.0 is available in Clear Linux since version **17890**.
+
+*Note:
+If you already have a previous Clear Linux version with Clear Containers 2.x installed,
+just update your OS to the latest version to get Clear Containers 3.0.*
+
+1. Run the `swupd` command to ensure the host is using the latest Clear Linux Version.
+
+```
+$ sudo swupd update
+```
+
+2. Install the Clear Containers bundle.
+
+```
+$ sudo swupd bundle-add containers-virt
+```
+
+3. Start the Docker\* and Clear Containers systemd services.
+
+```
+$ sudo systemctl enable docker
+$ sudo systemctl start docker
+$ sudo systemctl enable cc3-proxy
+$ sudo systemctl start cc3-proxy
+```
+
+4. Run Clear Containers 3.0.
+
+	You are now ready to run Clear Containers 3.0. For example:
+
+```
+$ sudo docker run -ti busybox sh
+```
+
+5. More information about Docker in Clear Linux.
```

Comment:
Because it is not a task/action, let's have this be a second level heading:
##More information...